packages/manager/apps/web/client/app/private-database/database/ram/update/private-database-database-ram-update.controller.js

```diff
@@ -96,39 +87,31 @@ angular.module('App').controller(
         : this.$translate.instant('price_ht_label', { price: price.text });
     }
 
-    getDurations() {
-      this.loading.durations = true;
+    getProvisionalOrder() {
+      this.loading.provisionalOrder = true;
 
       this.privateDatabaseService
-        .getRamPrices(this.productId, {
-          ram: this.model.capacity,
-        })
-        .then(
-          (durations) => {
-            this.loading.durations = false;
-            this.data.durations.available = durations;
-          },
-          angular.noop,
-          (duration) => {
-            this.data.durations.available = duration;
-          },
-        );
+        .getUpgradeProvisionalOrder(this.productId, this.model.planCode)
+        .then((response) => {
+          this.loading.provisionalOrder = false;
+          this.data.provisionalOrder = response;
+        });
```

Comment:
There should be a `catch` statement that alerts something went wrong when fetching
As a result `this.loading.provisionalOrder = false;` should move into a `finally` statement